packages/medusa-payment-paypal/src/types.ts

```diff
@@ -0,0 +1,35 @@
+export interface PaypalOptions {
+  /**
+   * Indicate if it should run as sandbox, default false
+   */
+  sandbox?: boolean
+  client_id: string
+  client_secret: string
+  auth_webhook_id: string
+  backend_url: string
```

Comment:
todo: As per our new convention, this should be camelCased. Though, fine to keep these for backward compatibility, if you find it to be a good idea.